SteamBot/SteamTradeDemoHandler.cs

```diff
@@ -202,9 +202,11 @@ public override void OnTradeSuccess()
         public override void OnTradeAwaitingEmailConfirmation(long tradeOfferID)
         {
             Log.Warn("Trade ended awaiting email confirmation");
-            SendChatMessage("Please complete the email confirmation to finish the trade");
+            SendChatMessage("Please complete the email confirmation to finish the trade, you've got 10 minutes from now.");
```

Comment:
The trade has been already approved by both sides at this point, the user has to go to their email and either click accept or decline. The bot can not actually cancel the trade at this point, so I've limited the time to 10 minutes, so it doesn't check periodically forever until the bot is shut down in case the user doesn't click on either button.